test/resources/nightly-config.yaml

```diff
@@ -0,0 +1,99 @@
+#!productName=RHBA
+#!groovyScriptsPath=https://groovyScriptsPathUrl
+#!datetimeSuffix=
+#!communityVersion=7.33.0
+#!installerCommonsVersion=2.3.6.rhba
+#!productVersion=
+#!izpackVersion=4.5.2.rhba-redhat-00006
+#!erraiVersion=4.7.0.Final
+#!mvelVersion=2.4.3.Final-redhat-1
+#!thorntailVersion=2.5.1.Final-redhat-00010
```

Comment:
do we need all those properties above?

Reply:
I think this file is used just for testing, so I suppose it can have whatever fields required?